WLC/code_fixing/codefixer.py

```diff
@@ -1,92 +1,181 @@
 import keyword
+import logging
+from string import digits
```

Comment:
Sorry for being so pedantic but can you reformat the import so that they are consistent with the overall style we have?

Reply:
Huh? Whats wrong with this? Do you disagree with the specific import?

Reply:
No, I am not talking about the imports. But having `import {}` together in a block and `from {} import {}` together in a block with empty lines between them.

Reply:
I was not aware this was the codestyle we were following - I thought we were following the splitting of standard imports, external imports, and relative imports, as this is the usual python way.

Reply:
There is no point is keeping this PR open because of this. We will talk about this in person.